### ISSUE TITLE
When exiting, NUSspli will say that it is taking a minute, and will crash. 

### ISSUE THREAD

**Issue by MrCurious10** [V10lator/NUSspli#376]
When I open NUSspli (Both Aroma and WUP installer versions), close out of it without going into any menus, just staying the NUSspli home menu, it will take a while to "Close" then crash. A  0.1 second part of the song will loop until Tha ends of time. You can not go home. You can't really do anything except hold the power button on the Wii U to make it do a Hard Shut Down. 
![20241127_185549](https://github.com/user-attachments/assets/eb0608a2-2367-42bd-a5c7-60e719e4bf3e)
Image of the crash happening.

**Comment by V10lator** [comment]
What Aroma plugins are you using?

**Comment by Un4ble974** [comment]
Hi all 

Got da same issue

**Comment by MrCurious10** [comment]
Me too.

**Comment by MrCurious10** [comment]
@V10lator, Yeah. The plug-ins I got are: Better Settings, Close/Restart Game, DRC Region Free, evWii, ftpiiu, Gamepad Volume Changer, GiveMiiYouTube, Homebrew On Menu, I Kay (Pretendo), re_nfpii, Region Free (A different one), Screenshot, Swip Swap Me, Wii U Time Sync, Wii VC Launch, Wii load Plugin, and of course, Aroma Base.

**Comment by V10lator** [comment]
Thanks @MrCurious10

Could you try without the Close/Restart Game and Wii U TIme Sync plugins, please? In case that works try readding one by one to see which one breaks. In case it doesn't work try without the 3rd party Region Free plugin, please.

Don't forget to report your findings even if nothing helps, please.

**Comment by MrCurious10** [comment]
Oh! My Bad! The 3rd party region free was actually DRC Region Free. I don't think I should get rid of that. 😅 But yeah. Even with both of those plug-ins gone, it still crashes.

**Comment by MrCurious10** [comment]
The only way to prevent the crash is either, 1) No NUSspli, or 2) Go into a menu, exit it, and then you can leave.

**Comment by MrCurious10** [comment]
Nevermind. This just in!!! Option 1 is the safest option.
